sim/Makefile

```diff
@@ -52,10 +52,7 @@ else
 endif
 
 lib:
-ifndef IPAPPROX
-	@$(VLIB) work
-	$(VMAP) work $(mkfile_path)/work
-else
```

Comment:
Hello @micprog, I think the rest of the code looks good but removing this seems to make the CI fails the simulations. Can you have a look into it?

Reply:
Hi @OttG,
Thanks for letting me know! My reason for removing these lines was because normally the work library is the default library, so linking is not strictly necessary. However, pulp-runtime will need direct access to this library (i.e. a symlink to the work directory), which was patched in a newer version. I have updated the runtime, and also restored the explicit library linking to restore previous functionality, I'll check the CI to make sure everything worked as intended.